src/components/Editor/index.js

```diff
@@ -442,6 +442,7 @@ class Editor extends PureComponent<Props, State> {
     // to flash the previously saved selected line.
     if (
       line !== null &&
+      this.lastJumpLine !== 0 &&
```

Comment:
if last jumpline was 0 it means we are jumping to the start of the document. and we don't want highlight.